stages/org.osbuild.selinux

```diff
@@ -4,26 +4,34 @@ import pathlib
 import sys
 
 import osbuild.api
-from osbuild.util import selinux
+from osbuild.util import parsing, selinux
 
 
-def main(tree, options):
+def main(args):
+    # Get the path where the tree is
+    tree = args["tree"]
+    options = args["options"]
     file_contexts = options.get("file_contexts")
     exclude_paths = options.get("exclude_paths")
+    target = options.get("target", "tree:///")
+    root, target = parsing.parse_location_into_parts(target, args)
 
     if file_contexts:
-        file_contexts = os.path.join(f"{tree}", options["file_contexts"])
+        if "://" not in file_contexts:
+            file_contexts = os.path.normpath(f"{tree}/{file_contexts}")
```

Comment:
Or just add `/` to `file_contexts` here when needed:
```
     if file_contexts:
-        if "://" not in file_contexts:
-            file_contexts = os.path.normpath(f"{tree}/{file_contexts}")
-        else:
-            file_contexts = parsing.parse_location(file_contexts, args)
+        if "://" not in file_contexts and not file_contexts.startswith("/"):
+            file_contexts = f"/{file_contexts}"
+        file_contexts = parsing.parse_location(file_contexts, args)
```

Reply:
This is the same situation as with implying `tree://`.  I'd like to g the opposite direction here, forcing bare paths (without a scheme) to be absolute when passed to `parse_location`, but it's hard to know how widespread it is arbitrary strings for paths are.

My preference would be to make it an error for paths to be relative (like it already is on `main`).  If we start using the function in more stages and scenarios where relative paths are used and we end up with a lot of rel-to-abs conversions everywhere, we can decide to change the function to accept relative paths.

Reply:
If i enforce now `file_contexts` to be abs path, than almost all json-/yaml- configs in repo would be broken. In other words this will be a breaking change. Leaing it as it is